### ISSUE TITLE
Bot checks every time

### ISSUE THREAD

**Issue by hubba900** [intika/Librefox#89]
Every time when I try to log into some sites (like tripadvisor) I should perform anti bot checks (choose traffic lighters, crossroads, etc). Is it proper way how to skip it? This is very annoyed.

**Comment by hubba900** [comment]
Could you provide settings for mozilla.cfg? I have comment lockPref("network.cookie.lifetimePolicy", 2), but this is not enough. :-(

**Comment by intika** [comment]
You changed the right setting :) but you need to change other things as well here is what to do to stay connected on sites and avoid bot check :

1. Comment `lockPref("network.cookie.lifetimePolicy", 2) ` by adding `//` at the beginning of the line under the file `mozilla.cfg`

2. Open Firefox's preferences, under "Privacy & Security" (about:preferences#privacy) right to `Clear history when Firefox closes` click `Settings...` and uncheck `Cookies` and `Active Logins `

**Comment by dimqua** [comment]
> if it detect other activities like amazon etc. (over cookies)

Is this technique require third-party cookies to be enabled?

**Comment by intika** [comment]
Yes and no... (https://browserleaks.com/social) if it's a service from google let say (youtube etc.) third party not required otherwise yes it is required

**Comment by Atavic** [comment]
*privacy.firstparty.isolate.restrict_opener_access* set to false lowers some isolation rules and can help those logging into Google or Facebook.

**Comment by Atavic** [comment]
Related issue: https://github.com/pyllyukko/user.js/issues/437
